### PR TITLE
Minor bug fixes in (collision) distance field

### PR DESCRIPTION
**Pull request by tsijs**
### Description

Made some minor bug fixes in the distance field functionality. 

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Decide if this should be cherry-picked to other current ROS branches

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by rhaschke** [comment]
@tsijs, please [apply clang-format](https://moveit.ros.org/documentation/contributing/code/) to fix Travis:
`find . -name '*.h' -or -name '*.hpp' -or -name '*.cpp' | xargs clang-format-3.9 -i -style=file`

**Comment by tsijs** [comment]
Done:)

File: moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h [hunk 1]
```diff
@@ -43,6 +43,7 @@
 #include <sstream>
 #include <memory>
 #include <float.h>
+#define protected public
```
Comment (henningkayser) [inline moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h]: @tsijs These changes are not safe and can lead to undefined behavior. If you want to expose certain protected fields please add getter/setter functions.

File: moveit_ros/benchmarks/examples/demo_panda_all_planners.yaml [hunk 2]
```diff
@@ -27,7 +27,4 @@ benchmark_config:
         - plugin: chomp_interface/CHOMPPlanner
           planners:
             - CHOMP
-        - plugin: stomp_moveit/StompPlannerManager
```
Comment (henningkayser) [inline moveit_ros/benchmarks/examples/demo_panda_all_planners.yaml]: Was this removed intentionally?

File: moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h (hunk 1 above)
Comment (tsijs) [reply moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h]: I know, I didnt mean to push this commit to the pull request:$

File: moveit_ros/benchmarks/examples/demo_panda_all_planners.yaml (hunk 2 above)
Comment (tsijs) [reply moveit_ros/benchmarks/examples/demo_panda_all_planners.yaml]: See other remark

**Comment by rhaschke** [comment]
@henningkayser, can you have a look (again)?

**Comment by henningkayser** [comment]
@tsijs, @rhaschke do you want this cherry-picked to melodic?

**Comment by rhaschke** [comment]
> Do you want this cherry-picked to melodic?

Strictly speaking, this changes public API (from Vector3i to Vector3d). However, as this is a clear bug fix, I support cherry-picking this. @henningkayser, please, in future, clean up the commit message, when squash-merging on github.

**Pull request merged**